pkg/playback/server_test.go

```diff
@@ -17,8 +17,8 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-const defaultLocalAddress = "localhost:8080"
-const defaultLocalWebhookAddress = "localhost:8888"
+const defaultLocalAddress = "127.0.0.1:8080"
+const defaultLocalWebhookAddress = "127.0.0.1:8888"
```

Comment:
There was a bug that made `make test` fail unless I changed this, see https://github.com/golang/go/issues/41572#issuecomment-700232101